### DISCUSSION TITLE
Noob IPAM Setup Question - Searching IP returns parent prefix / network

### DISCUSSION THREAD

**Discussion by wm-cianphillips** [netbox-community/netbox#14556] [General]
Hi there,

I'm new to Netbox and trying to get the IPAM portion configured so that we can easily find what network / vlan an IP address belongs to. We've entered all of the networks as prefixes, but searching on an IP does not return the prefix it belongs to. Should we be entering the ranges instead of or in addition to the prefixes?

I searched for a similar topic but didn't find anything related. Perhaps the question is too basic.

**Comment by candlerb** [comment]
Search doesn't necessarily work how you might expect. Old issues: #6016, #1410

> searching on an IP does not return the prefix it belongs to

Can you give a more specific example? And what version of Netbox are you using?

With v3.6.7, if I use the global search box and search for "10.12.255.61" (which takes me to `/search/?q=10.12.255.61`) then I *do* find prefixes which enclose that address:

![image](https://github.com/netbox-community/netbox/assets/44789/3f465ba9-7c9f-4976-9f90-506c5fda2cb2)
